libs-scala/grpc-utils/BUILD.bazel

```diff
@@ -4,17 +4,38 @@
 load(
     "//bazel_tools:scala.bzl",
     "da_scala_library",
+    "da_scala_test_suite",
 )
 
 da_scala_library(
     name = "grpc-utils",
     srcs = glob(["src/main/scala/**/*.scala"]),
     tags = ["maven_coordinates=com.daml:grpc-utils:__VERSION__"],
+    versioned_scala_deps = {
+        "2.12": ["@maven//:org_scala_lang_modules_scala_collection_compat"],
```

Comment:
Wasted some time due to this, as initially, I had specified this as a normal scala_dep. Turns out that there are places where this is not specified as a versioned one and it still works. 😕